app/src/main/java/org/wikipedia/gallery/GalleryActivity.java

```diff
@@ -295,8 +296,10 @@ protected void setTheme() {
     public void onActivityResult(int requestCode, int resultCode, final Intent data) {
         super.onActivityResult(requestCode, resultCode, data);
         if (requestCode == ACTIVITY_REQUEST_DESCRIPTION_EDIT && resultCode == RESULT_OK) {
-            FeedbackUtil.showMessage(this, getString(R.string.description_edit_success_saved_image_caption_in_lang_snackbar,
-                    app.language().getAppLanguageLocalizedName(StringUtils.defaultString(targetLanguageCode, app.language().getAppLanguageCode()))));
+            if (!Prefs.shouldShowSuggestedEditsSurvey()) {
```

Comment:
I think we should not hide the Snackbar here since the survey Snackbar will only show on the Explore feed.